File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.py
def solution(s):
    print(len(s))
    if (len(s) == 4) | (len(s) == 6):
        n = [x for x in s if ((x) >= '0') & ((x) <= '9')]
        if len(s) == len(n):
            return True
        else:
            return False
    else:
        return False

File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.swift
import Foundation
func solution(_ s:String) -> Bool {
    var answer : Bool = false
    let stringArray : Array = Array(s)
    var index : Int = 0
    if ((stringArray.count == 4) || (stringArray.count == 6)) {
        for array in stringArray {
            if (array >= "0") && (array <= "9") {
                index += 1
            }
        }
        if index == stringArray.count {
            answer = true
        }else{
            answer = false
        }
    }else{
        answer = false
    }
    return answer
}